sql/firefox_nondesktop_exact_mau28_by_product_by_dimensions.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  -- 2019 goals define MAU for a given observation date based on a window of the
+  -- previous 28 days, not including the observation date. To be consistent with goals,
+  -- users should plot based on observation_date rather than submission_date.
+  DATE_ADD(submission_date, INTERVAL 1 DAY) AS observation_date,
+  submission_date AS last_submission_date_in_window,
+  * EXCEPT (submission_date, generated_time, normalized_channel)
+FROM
+  firefox_nondesktop_exact_mau28_raw_v1
```

Comment:
should the raw here be used more widely?

Reply:
Wherever a table is really only intended as a step towards another view (materialized or live), `raw` seems like a useful suffix.